### DISCUSSION TITLE
Essential hub does not charge

### DISCUSSION THREAD

**Discussion by ojuuji** [pybricks/support#1777] [Q&A]
I used Pybricks on Essential hub more than a year ago for my MOC, and hub has been unused since then. Now when connecting USB cable, hub turns on but does not charge (I left it for 12 hours to ensure). The charging LED blinks with red and green, and hub makes high pitch noise. I tried to remove battery and plug it back - nothing changed.

When connected to USB it can be turned on and off, and Pybricks Code can connect to it. Here is `print(pybricks.version)` output:

```
('essentialhub', '3.2.1', 'v3.2.1 on 2022-12-26')
```

However when disconnecting USB it turns off immediately and does not turn on when pressing or holding the button. So I cannot restore official firmware nor update Pybricks firmware.

Is there anything I can do with it?

**Comment by laurensvalk** [comment]
This is a well known issue with the essential hub battery, also with the stock firmware. I'll try adding some links to other known cases tomorrow.

LEGO customer service usually replaces the battery at no charge, especially since it is a well known problem.

As a side note - I just received one such bad battery, so I'll have a closer look at it to see if there is a way to trickle charge them.
